cmd/clusterctl/client/config/providers_client.go

```diff
@@ -37,6 +37,7 @@ const (
 
 // Infra providers.
 const (
+	AkamaiProviderName         = "linode-linode"
```

Comment:
```suggestion
	LinodeProviderName         = "linode-linode"
```

Let's maybe call this linode :-) because it targets linode cloud, like aws provider which is not maintained by aws but targets aws

Reply:
good catch, I've now removed all references to Akamai in the go code, it should be standardized on `Linode`